### PR TITLE
refactor: change bridge call

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
Stop using local bridge variable to avoid early injection.

## Links
MINI-2148

# Checklist
- [ ] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

File: js-miniapp-sdk/src/miniapp.ts [hunk 1]
```diff
@@ -118,36 +120,28 @@ export interface UserInfoProvider {
...
-  }
-
   getUserName(): Promise<string> {
-    return this.bridge.getUserName();
+    return (window as any).MiniAppBridge.getUserName();
   }
 
   getProfilePhoto(): Promise<string> {
-    return this.bridge.getProfilePhoto();
+    return (window as any).MiniAppBridge.getProfilePhoto();
```
Comment (corycaywood) [inline js-miniapp-sdk/src/miniapp.ts:128]: Using `any` will make us lose the typing benefits of TypeScript, so we don't know whether we're accidentally trying to use a function on `MiniAppBridge` which doesn't exist. 

Could you instead create a function which dynamically returns the typed `MiniAppBridge`? For example:

```
function getBridge() {
  return (window as any).MiniAppBridge as MiniAppBridge;
}
```

Comment (minh-rakuten) [reply js-miniapp-sdk/src/miniapp.ts]: Yes, I have updated.

**Review by corycaywood — APPROVED** [review]
LGTM 👍

**Review by khairul-alam-licon — APPROVED** [review]
👍 👍

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**